dataland-backend/src/main/kotlin/org/dataland/datalandbackend/model/EuTaxonomyDataForFinancials.kt

```diff
@@ -0,0 +1,70 @@
+package org.dataland.datalandbackend.model
+
+import com.fasterxml.jackson.annotation.JsonProperty
+import org.dataland.datalandbackend.annotations.DataType
+import org.dataland.datalandbackend.model.enums.eutaxonomy.AttestationOptions
+import org.dataland.datalandbackend.model.enums.eutaxonomy.FinancialServicesType
+import org.dataland.datalandbackend.model.enums.eutaxonomy.YesNo
+import java.math.BigDecimal
+
+/**
+ * --- API model ---
+ * Fields of the questionnaire for EU-Taxonomy data for financial companies
+ */
+@DataType
+data class EuTaxonomyDataForFinancials(
+    @field:JsonProperty("financialServicesType", required = true)
+    val financialServicesType: FinancialServicesType?,
+
+    @field:JsonProperty("eligibilityKpis")
```

Comment:
Der Stil in diesem model weicht deutlich von den anderen ab. Ich würde es begrüßen wenn es einen Konsistenten Stil gibt. Können wir auch gerne im Team diskutieren welche Variante bevorzugt wird.

Reply:
Habe ich gerade geändert. Schaus dir gerne nochmal an